src/app/carbon-estimation-table/carbon-estimation-table.component.html

```diff
@@ -0,0 +1,81 @@
+<!-- eslint-disable @angular-eslint/template/interactive-supports-focus -->
+<table role="treegrid" aria-label="Estimation of emissions" class="tce-table-fixed tce-w-full">
+  <thead>
+    <tr class="tce-border-b-2">
+      <th class="tce-text-left">Category</th>
+      <th class="tce-w-40 tce-text-left">Emissions</th>
+    </tr>
+  </thead>
+  @if (carbonEstimation()) {
```

Comment:
Worth checking `tableData().length > 0` instead?

Reply:
Yeah it could. Will update as think it makes more sense as that is the table and tableData